### DISCUSSION TITLE
Table组件列显示/隐藏增加缓存

### DISCUSSION THREAD

**Discussion by caizya** [dotnetcore/BootstrapBlazor#4824] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the solution you'd like

麻烦大佬们Table组件的列显示/隐藏能否像列拖动一样增加缓存效果

**Comment by bb-auto** [comment]
@caizya Thank you for contacting us. We will give feedback later.

**Comment by ArgoZhang** [comment]
@caizya 请详细说明你的方案或者预期，仅仅一句话我们实在无法正确理解其含义，建议提交 PR

**Reply by caizya** [reply]
不好意思，我描述的不清楚，我预期的效果是列选择显示或隐藏后可以本地缓存起来，界面刷新或关闭浏览器后重新进入该界面按缓存的显示隐藏列渲染
![image](https://github.com/user-attachments/assets/63e80ca0-0cb1-44cb-a06e-a244a5978b5a)
例如图的地址列，设置了不显示，当前效果是刷新或重新进入界面后还会再显示，我的预期效果是不显示
